docs/concepts/endpoint.md

```diff
@@ -1,117 +1,137 @@
 ---
 id: endpoint
-title: Endpoint
+title: Endpoints in ZIO HTTP
 ---
```

Comment:
render the docs and see the sidebar. It might not be what you expect!

Reply:
![Uploading Screenshot 2024-04-22 at 13-35-48 Endpoints in ZIO HTTP zio-http-docs-website.png…]()

Reply:
i guess am leaving it with just `Endpoint`